Add bypass_disruptions Link

AzHicham opened the pull request:
Add bypass_disruptions Link if at least one disruption has a NO_SERVICE severity

Commit:
add link to realtime

Commit:
added test and change link-name

Commit:
fix possible bug

kadhikari reviewed (commented):
Can you add a test to verify if links is present in /jouneys probably in test file: https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/tests/chaos_disruptions_tests.py

pbench commented on source/jormungandr/jormungandr/scenarios/simple.py:
```diff
@@ -346,6 +346,13 @@ def _add_ridesharing_link(self, resp, params, instance):
+        # find first impact wit a NO_SERVICE severity
+        found = next((True for impact in resp.impacts if impact.severity.effect == Severity.NO_SERVICE), False)
+        if found:
+            params['data_freshness'] = 'realtime'
+            add_link(resp, rel='bypass_disruptions', **params)
```
A suggestion to avoid mutating `params` :
```suggestion
            cloned_params = copy.deepcopy(params)
            cloned_params['data_freshness'] = 'realtime'
            add_link(resp, rel='bypass_disruptions', **cloned_params)
```

kadhikari replied:
:+1:

AzHicham replied:
Thanks !

pbench reviewed (approved):
A small nitpicking, feel free to dismiss it.

kadhikari commented:
@AzHicham i added do_not_merge and ok_to_review while waiting for the tests

Commit:
added tests for bypass_disruptions links

kadhikari reviewed (approved):
A complete PR. :+1:

Commit:
Use deepcopy

Thanks to deepcopy we avoid any mistake when using _add_bypass_disruptions_link
